src/Asserts.ts

```diff
@@ -86,5 +86,40 @@ module OpenWeatherJS {
 				throw new TypeError(message);
 			}
 		}
+
+		/**
+		 * Validates provided value is a URL and throws a new TypeError
+		 * with specified message otherwise..
+		 *
+		 * @param value   - a value being tested.
+		 * @param message - a short description of the assertion.
+		 */
+		static isUrl(value: string, message: string): void {
+			var yourRegularExpression = /(^|\s)((https?:\/\/)?[\w-]+(\.[\w-]+)+\.?(:\d+)?(\/\S*)?)/gi;
```

Comment:
Can you please could it something differently to provide more information about this regular expression?